packages/website/posts/2022-01-25-what-am-buying.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'When you buy an NFT, what are you actually getting?'
+description: Behind the glamour, NFTs are more like hyperlinks than like files.
+author: J Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/87873179/146266410-dac03a35-dff7-461b-8096-254f04e5df04.jpg
+date: Jan 25, 2022
+tags:
+  - culture
+---
+
+When you buy an NFT, you are purchasing your name at the top of the list of holders. The value comes from the previous holders (and the link to the creator), more than the image. You are also buying the right to decide who is the next token holder. You could sell it or give it away.
```

Comment:
> more than the image

Not always an image! Could be a video, music, text, etc. "asset" maybe?

Note: you refer to "the image" throughout...

Reply:
good point